config/routes.rb

```diff
@@ -49,6 +49,10 @@
   get "video" => "welcome#video"
   get "vote-for-us" => "welcome#nycbigapps"
 
+  namespace :admin do
+    resources :buildings
```

Comment:
Ooooh, okay, that makes sense to me. Since it was all resources for buildings, show page included, I figured you were thinking that was only for admins. If there are other routes that present the data, then I'm fine with this.